app/views/content/landing/still-time-to-apply/_content.html.erb

```diff
@@ -0,0 +1,62 @@
+<div class="row">
+  <div class="grey col">
+    <section class="col col-720">
+      <p class="inset">
+        Every day, people are being accepted onto teacher training courses starting this September. Places fill up quickly at this time of year so don’t miss your opportunity to join a course while there’s still time to apply.
+      </p>
+<div class="row">
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Find a teacher training course</h2>
+      <p>
+      Teacher training courses usually take 9 months full-time, or 18 to 24 months part time.
+      </p>
+      <p>
+      Most courses involve school placements with some theoretical learning.
+      </p>
+      <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses">Find postgraduate teacher training.</a>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Get help to apply on time</h2>
+
+    <div class="inset">
+      <p>
+        Our teacher training advisers can save you time on research and guide you through everything you need to know to apply for teacher training.
+      </p>
+
+      <p>They can help you with:</p>
+
+      <ul>
+        <li>creating a personal statement that shows off your strengths</li>
+        <li>preparing for interviews</li>
+      </ul>
+
+      <p><%= link_to("Get a teacher training adviser", page_path("teacher-training-advisers")) %>.</p>
+    </div>
+  </section>
+
+    <section class="col col-720">
+
+    <h2 class="heading--box-blue">Applications from non-UK citizens</h2>
+
+    <div class="inset">
+      <p>
+        If you’re a non-UK citizen, the application process takes longer so not all providers will accept you onto a course at this stage.
+      </p>
+      <p>Learn more about <%= link_to("training to teach in England as a non-UK citizen", page_path("non-uk-teachers/train-to-teach-in-england-as-an-international-student")) %>.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+
+    <h2 class="heading--box-blue">If you need more time</h2>
+
+    <div class="inset">
+      <p>
+        Don’t worry if you’re not quite ready to apply yet. Applications will open in October for next year’s courses.
+      </p>
+      <p>To get a reminder nearer the time, <%= link_to("sign up for our mailing list", page_path("mailinglist/signup/name")) %>.</p>
```

Comment:
@philspray @EllieNodder We usually avoid saying 'mailing list' externally - the only other place it appears on the site is on the maths page (and I've always meant to amend it!). Can we change to something more in line with the mailing list proposition?

Reply:
any chance we could/would want to try a multi-variant test here and try out different mailing list CTAs? would we get enough traffic? could be interesting to compare with mailing list CTA testing we're planning on the other landing page @gemmadallmandfe @philspray 

(yes, I am now testing mad)